Sources/TSCBasic/Path.swift

```diff
@@ -247,6 +247,10 @@ public struct RelativePath: Hashable {
         _impl = impl
     }
 
+    internal init(string: String) {
```

Comment:
does this need to be internal?

Reply:
also should we give it a better name, eg static method that points out it is unsafe etc?

Reply:
I think that we can get away with `fileprivate`.  I'll change that.  If you mean, should it be public?  I'm not sure if we want to encourage people to make unsafe paths.

I'm not sure I understand the bit on the "static method".  However, I can adjust this to something like `init(unsafeUncheckedPath:)`, though I'm not particularly happy with that the label.  If you have suggestions about the label, please share them.

Reply:
We may have to of course revisit that as we get further through the test suite in SPM (I think that I'm only ~80% through it).

Reply:
fileprivate and new spelling is good to address the concern